calendar-aggregator-test/pom.xml

```diff
@@ -119,6 +83,9 @@
                         <goals>
                             <goal>stop</goal>
                         </goals>
+                        <configuration>
```

Comment:
I was getting the following error: `Spring application lifecycle JMX bean not found (fork is null). Could not stop application gracefully`, so I enabled the fork.